components/authz-service/testhelpers/testhelpers.go

```diff
@@ -233,8 +233,13 @@ func SetupTestDB(t *testing.T) (storage.Storage, *TestDB, *prng.Prng, *migration
 }
 
 func (d *TestDB) Flush(t *testing.T) {
-	_, err := d.Exec(`DELETE FROM iam_policies CASCADE; DELETE FROM iam_members CASCADE;
-		DELETE FROM iam_roles CASCADE; DELETE FROM iam_projects CASCADE; DELETE FROM iam_role_projects CASCADE;`)
+	_, err := d.Exec(`DELETE FROM iam_policies CASCADE;
+		DELETE FROM iam_members CASCADE;
+		DELETE FROM iam_roles CASCADE;
+		DELETE FROM iam_role_projects CASCADE;
+		DELETE FROM iam_project_rules;
+		DELETE FROM iam_staged_project_rules;
```

Comment:
Why is `CASCADE` not needed on the two project rule tables?

Reply:
Oversight; also, I've just added these to see if it makes tests pass. I'll pick this up properly today, I hope.

Reply:
Don't think there is anything that actually would get cascade deleted via those tables atm anyway but I can add them in.

Reply:
My question is why do we need those at all? They should be getting cascade deleted via `iam_projects`.

Reply:
The lack of cascade deletion was what was making integration tests fail :) Added to the FK constraints.